src/main/scala/devices/uart/UART.scala

```diff
@@ -104,46 +104,38 @@ class UARTRx(c: UARTParams)(implicit p: Parameters) extends UARTModule(c)(p) {
   val debounce_max = (debounce === UInt(3))
   val debounce_min = (debounce === UInt(0))
 
-  val prescaler = Reg(init = UInt(0, uartDivisorBits - uartOversample))
+  val prescaler = Reg(UInt(width = uartDivisorBits - uartOversample + 1))
```

Comment:
is there a reason to change the behavior s.t. this register is no longer reset?

Reply:
The state machine no longer depends on the `prescalar` register having a defined reset value - it is always overwritten on the transition to the `s_data` state.